apps/io500/io500.pbs

```diff
@@ -0,0 +1,157 @@
+NSTRUCTIONS:
+# Edit this file as needed for your machine.
+# It is a simplified version of the site-configs/sandia/startup.sh which include SLURM directives.
+# Most of the variables set in here are needed for io500_fixed.sh which gets sourced at the end of this.
+# Please also edit 'extra_description' function.
+
+FILESYSTEM=/beegfs
```

Comment:
please consider this as a script parameter so ut can be independant from the FS to test

Reply:
FILESYSTEM is now passed in as a PBS variable (-v option).